src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkDiffExclusions.txt

```diff
@@ -0,0 +1,29 @@
+# This list is processed using FileSystemName.MatchesSimpleExpression
+#
+# Format
+# {msft|sb},<path> [# comment]
+# msft = Microsoft built SDK
+# sb   = source-built SDK
+#
+# Examples
+# 'folder/*' matches 'folder/' and 'folder/abc'
+# 'folder/?*' matches 'folder/abc' but not 'folder/'
+#
+# We do not want to filter-out folder entries, therefore, we should use: '?*' and not just '*'
+
+msft,./sdk/x.y.z/TestHostNetFramework/?*   # Intentional - MSFT build includes test-host that targets netcoreapp3.1
+msft,./sdk/x.y.z/Sdks/Microsoft.NET.Sdk.Publish/tools/net472/?*   # Intentional - source-build includes SDK Publishing package that target latest .NET TFM
+msft,./sdk/x.y.z/Sdks/Microsoft.NET.Sdk.WindowsDesktop/?*   # Intentional - explicitly excluded from source-build
+sb,./sdk/x.y.z/TestHost/?*   # Intentional - source-build includes test-host that targets latest .NET TFM
+
+# vstest localization is disabled in Linux builds - https://github.com/microsoft/vstest/issues/4305
+msft,./sdk/x.y.z/*?/Microsoft.TestPlatform.*?.resources.dll
+msft,./sdk/x.y.z/*?/Microsoft.VisualStudio.TestPlatform.*?.resources.dll
+msft,./sdk/x.y.z/*?/Test.Utility.resources.dll
+msft,./sdk/x.y.z/*?/vstest.console.resources.dll
+msft,./sdk/x.y.z/Extensions/*?/Microsoft.TestPlatform.*?.resources.dll
+msft,./sdk/x.y.z/Extensions/*?/Microsoft.VisualStudio.TestPlatform.*?.resources.dll
+
+# nuget localization is not available for Linux builds - https://github.com/NuGet/Home/issues/12440
+msft,./sdk/x.y.z/*?/NuGet.*?.resources.dll
+msft,./sdk/x.y.z/Sdks/NuGet.Build.Tasks.Pack/*?/NuGet.*?.resources.dll
```

Comment:
I have a few questions about the remaining diffs.  

1. Would it be appropriate to add an exclusion for the known diffs that are caused by ilmerge not being supported in source-build?
2. The following differences are caused by https://github.com/dotnet/source-build/issues/3242 and should be added as exclusions IMO.
```
-./sdk-manifests/x.y.z/microsoft.net.sdk.android/
-./sdk-manifests/x.y.z/microsoft.net.sdk.android/WorkloadManifest.json
-./sdk-manifests/x.y.z/microsoft.net.sdk.android/WorkloadManifest.targets
-./sdk-manifests/x.y.z/microsoft.net.sdk.ios/
-./sdk-manifests/x.y.z/microsoft.net.sdk.ios/WorkloadManifest.json
-./sdk-manifests/x.y.z/microsoft.net.sdk.ios/WorkloadManifest.targets
-./sdk-manifests/x.y.z/microsoft.net.sdk.maccatalyst/
-./sdk-manifests/x.y.z/microsoft.net.sdk.maccatalyst/WorkloadManifest.json
-./sdk-manifests/x.y.z/microsoft.net.sdk.maccatalyst/WorkloadManifest.targets
-./sdk-manifests/x.y.z/microsoft.net.sdk.macos/
-./sdk-manifests/x.y.z/microsoft.net.sdk.macos/WorkloadManifest.json
-./sdk-manifests/x.y.z/microsoft.net.sdk.macos/WorkloadManifest.targets
-./sdk-manifests/x.y.z/microsoft.net.sdk.maui/
-./sdk-manifests/x.y.z/microsoft.net.sdk.maui/WorkloadManifest.json
-./sdk-manifests/x.y.z/microsoft.net.sdk.maui/WorkloadManifest.targets
-./sdk-manifests/x.y.z/microsoft.net.sdk.tvos/
-./sdk-manifests/x.y.z/microsoft.net.sdk.tvos/WorkloadManifest.json
-./sdk-manifests/x.y.z/microsoft.net.sdk.tvos/WorkloadManifest.targets
```
3. Do you have an understanding of what the cause is of the following two resource.dll diffs are?
./sdk/x.y.z/cs/Microsoft.Build.NuGetSdkResolver.resources.dll

If you don't know now that is fine.  I think issues should be logged if they don't already exist for the remaining entries in the baseline.
./sdk/x.y.z/cs/Microsoft.CodeCoverage.IO.resources.dll

Reply:
I'll address 1st and 2nd points. I'm not 100% sure if 3rd is related to NuGet-repo localization - will check and include it if it is - same for Microsoft.CodeCoverage.IO.resources.dll.